### PR TITLE
{bio}[foss/2023a] HOMER v4.11 w/ R 4.3.2, libidn2 v2.3.7, wget v1.24.5

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/h/HOMER/HOMER-4.11-foss-2023a-R-4.3.2.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+checksums = ['ccdaa3004a0e0df0882634671d4a1acc88364761e0e6c7ea329ebbf1eb729537']
+
+dependencies = [
+    ('Perl', '5.36.1'),
+    ('R', '4.3.2'),
+    ('SAMtools', '1.18'),
+    ('R-bundle-Bioconductor', '3.18', versionsuffix)
+]
+
+postinstallcmds = ["cd %(installdir)s && perl ./configureHomer.pl -install -all"]
```
Comment (boegel) [inline easybuild/easyconfigs/h/HOMER/HOMER-4.11-foss-2023a-R-4.3.2.eb:27]: Make sure the specific version is installed, as is done in https://github.com/bear-rsg/easybuild-easyconfigs/blob/2020a/easybuild/easyconfigs/h/HOMER/HOMER-4.11-foss-2020a.eb

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@PetrKralCZ The failure on `jsc-zen3` is because `wget` is not available, we should include that as a build dependency:

```
== 2024-05-23 09:28:59,436 run.py:687 INFO cmd "cd /tmp/boegelbot/pr20590/software/HOMER/4.11-foss-2023a-R-4.3.2 && perl ./configureHomer.pl -install -all" exited with exit code 0 and output:

        Current base directory for HOMER is /tmp/boegelbot/pr20590/software/HOMER/4.11-foss-2023a-R-4.3.2/./

sh: line 1: wget: command not found
        Updating Settings...
        If updating ALL, the following needs to be updated:
                SOFTWARE
                ORGANISMS
                PROMOTERS
                GENOMES

`wget -O 0.874620091010915.tmp http://homer.ucsd.edu/homer/configureHomer.pl`;
sh: line 1: wget: command not found
diff: 0.874620091010915.tmp: No such file or directory
        configureHomer.pl script is up-to-date
rm: cannot remove '0.874620091010915.tmp': No such file or directory
        Packages to Install...
                homer -> !!! Could not find homer (SOFTWARE)
        Pausing for 3 seconds (hit ctrl-C to stop)
Trouble opening /tmp/boegelbot/pr20590/software/HOMER/4.11-foss-2023a-R-4.3.2/.//bin/findMotifs.pl
readline() on closed filehandle IN at ./configureHomer.pl line 863.
mv: cannot move '.new.pl' to '/tmp/boegelbot/pr20590/software/HOMER/4.11-foss-2023a-R-4.3.2/.//bin/findMotifs.pl': No such file or directory
chmod: cannot access '/tmp/boegelbot/pr20590/software/HOMER/4.11-foss-2023a-R-4.3.2/.//bin/findMotifs.pl': No such file or directory
...
```

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
@PetrKralCZ Last build failure shows:
```
        Checking for wget... /project/def-maintainers/boegelbot/rocky9/zen3/software/wget/1.24.5-GCCcore-12.3.0/bin/wget
        Checking for cut... /usr/bin/cut
        Checking for gcc... /project/def-maintainers/boegelbot/rocky9/zen3/software/GCCcore/12.3.0/bin/gcc
        Checking for g++... /project/def-maintainers/boegelbot/rocky9/zen3/software/GCCcore/12.3.0/bin/g++
        Checking for zip... which: no zip in (...)
        The program zip was not found but is strictly required
        Checking for unzip... /usr/bin/unzip
        Checking for make... /usr/bin/make
        Checking for tar... /usr/bin/tar
        Checking for gunzip... /project/def-maintainers/boegelbot/rocky9/zen3/software/gzip/1.12-GCCcore-12.3.0/bin/gunzip
        Checking for gzip... /project/def-maintainers/boegelbot/rocky9/zen3/software/gzip/1.12-GCCcore-12.3.0/bin/gzip

        Installation will halt since one or more required programs where not found.
                (or they are not in your executable path)
        Most of these programs are standard Unix Utilities that should be installed by default.
        If you are running Mac OS X, make sure to install developer tools

        Also, if you think these programs have already been installed, make sure you modify your
        configuration files to include each progam in the executable path (i.e. ~/.bash_profile)
        Refer to the Homer documentation if you need more help: http://homer.ucsd.edu/homer/

        --- Cannot Continue until required software is available ---
```

So you should also add `Zip` and `UnZip` as (build) depenencies.
For `cut`, `make`, `tar` it's typically fine to rely on OS.

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3120.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/ed9ab082414cd5daf036b8d7d339f284 for a full test report.

**Review by boegel — APPROVED** [review]

**Pull request merged**